InventoryInStorePickupFrontend/view/frontend/web/js/view/shipping-information/address-renderer/store-pickup-address.js

```diff
@@ -1,71 +0,0 @@
-/**
```

Comment:
Looks like we can remove `InventoryInStorePickupFrontend/view/frontend/web/template/shipping-information/address-renderer/store-pickup-address.html` as well

Reply:
Done.